2009-02-28  Török Edwin  <edwintorok@gmail.com>  (tiny change)
	    Ralf Wildenhues  <Ralf.Wildenhues@gmx.de>

	Do not add bogus directory arguments to link command lines.
	* libltdl/config/ltmain.m4sh (func_mode_link): Ensure $path is
	always initialized before it is used.  Reported for zsh, for
	which $path contains $PATH entries even after "emulate sh", see
	<http://bugs.debian.org/517501>.

Index: libtool-2.2.6a/libltdl/config/ltmain.m4sh
===================================================================
--- libtool-2.2.6a.orig/libltdl/config/ltmain.m4sh	2009-04-07 19:48:31.000000000 +0200
+++ libtool-2.2.6a/libltdl/config/ltmain.m4sh	2009-04-07 23:04:27.000000000 +0200
@@ -5454,6 +5454,7 @@
 	  if test "$link_all_deplibs" != no; then
 	    # Add the search paths of all dependency libraries
 	    for deplib in $dependency_libs; do
+	      path=
 	      case $deplib in
 	      -L*) path="$deplib" ;;
 	      *.la)
